extensions/retool/src/utils.ts

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useMemo, useState } from "react";
+import algoliasearch from "algoliasearch/lite";
+
+const APPID = "T28YKFATPY";
+const APIKEY =
+  "YWY1ZGRlZGY5M2I4YTkxZDg3Y2FiOWYyODc5M2RkMGMxNzQ0NzM2NmU5ZjFjMDI0Y2M2YmFjNTVjYTUzNjJmZHRhZ0ZpbHRlcnM9KHByb2plY3Q6NTljMDkxZDk5MGRlMzMwMDEwNzZiMzI5KSwodmVyc2lvbjpub25lLHZlcnNpb246NjI0ZjE3YWI1NjYwYzEwOWQ5MDUwMjRlKSwoaGlkZGVuOm5vbmUsaGlkZGVuOmZhbHNlKSwoaW5kZXg6Q3VzdG9tUGFnZSxpbmRleDpQYWdlKQ==";
```

Comment:
This probably should be something the user passes in

Reply:
Why? I grabbed these keys from the browser console. They are the same for anyone who reads the Retool docs on the web. Asking people to add these keys would not be really user friendly imo.